build/debian.sh

```diff
@@ -1,48 +0,0 @@
-#!/bin/bash
```

Comment:
don't remove the whole file... I was just asking you to remove the changes from the file

You can download this copy of the file into your branch:
```cd build && wget https://raw.githubusercontent.com/resiprocate/resiprocate/master/build/debian.sh```

and then ```git add``` it and it will be the same as before.